plugins/HydrogenImport/LocalFileMng.h

```diff
@@ -14,20 +14,13 @@ namespace lmms
 class LocalFileMng
 {
 public:
-	LocalFileMng();
-	~LocalFileMng();
-	std::vector<QString> getallPatternList(){
```

Comment:
Is this an unused variable?

Reply:
Yeah, this is only ever used as a static class.